nixos/tests/atticd.nix

```
@@ -29,6 +29,40 @@ in
       ];
     };
 
+    localWithPostgres = {
+      services.atticd = {
+        enable = true;
+        settings = {
+          database = {
+            url = "postgres:///atticd?host=/run/postgresql";
+          };
+        };
+
+        inherit environmentFile;
+      };
+
+      services.postgresql = {
+        enable = true;
+
+        ensureDatabases = [ "atticd" ];
+        ensureUsers = [
+          {
+            name = "atticd";
+            ensureDBOwnership = true;
+          }
+        ];
+      };
+
+      systemd.services.atticd.serviceConfig = {
```

I copy&pasted this from another test. Looks like it was wrong there.

```suggestion
      systemd.services.atticd.unitConfig = {
```